elements/bulbs-video/components/revealed.test.js

```diff
@@ -596,6 +596,7 @@ describe('<bulbs-video> <Revealed>', () => {
         utmTestData = {
           utmSource: 'test source',
           utmCampaign: 'test campaign',
+          utmMedium: 'test medium',
```

Comment:
search this file for `utm_campaign`, I see a few places where you probably want to add similar checks for `utm_medium` too.